core/spec/models/spree/concerns/ransackable_attributes_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class DummyClass < Spree::Base
+end
+
+RSpec.describe Spree::RansackableAttributes do
+  let(:test_class) { DummyClass }
```

Comment:
We can use `Class.new(Spree::Base)` so that we don't leak a `DummyClass` top-level constant in other tests.

Reply:
Great suggestion, thank you!